ui/src/components/questions/QuestionItem.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {Typography} from "@material-ui/core";
+import Grid from "@material-ui/core/Grid";
+import Avatar from "@material-ui/core/Avatar";
+import "../../css/App.css";
+import Button from "@material-ui/core/Button";
+
+export const QuestionItem = (props) => {
+
+    let answerText = props.questionItem.answer.split('\n').map((item, i) => {
```

Comment:
Rende Raw html.

Reply:
Couldn't solve it fast, will make it in seaprate PR